.gitignore

```diff
@@ -40,6 +40,9 @@ phpunit.xml
 .phpunit.result.cache
 .phpunit.cache
 
+# phpstan
+phpstan.neon
+
```

Comment:
we use the `phpstan.dist.neon` which is commited now for local overrides the phpstan.neon can be used.